gnomad_qc/v5/data_ingestion/federated_validity_checks.py

```diff
@@ -306,6 +566,24 @@ def main(args):
             verbose=verbose,
         )
 
+        handler.flush()
+        log_output = log_stream.getvalue()
+
+        # TODO: Create resource functions when know organization of federated data.
+        log_file = (
+            "gs://gnomad-tmp/federated_validity_checks/federated_validity_checks.log"
+        )
+        output_file = (
+            "gs://gnomad-tmp/federated_validity_checks/federated_validity_checks.html"
+        )
```

Comment:
Agree with a resource function or some standardized format, maybe proejct name and date in the file --  but for now, want to make the output file a passed arg that defaults to this? `gs://gnomad-tmp/federated_validity_checks/federated_validity_checks` or something?

Reply:
yup will make it an arg for now